docs/quick-start-k8s.rst

```diff
@@ -0,0 +1,428 @@
+
+Add the NVIDIA NGC Helm repository:
+
+.. code-block:: bash
+
+   helm repo add nvidia https://helm.ngc.nvidia.com/nvidia
+   helm repo update
+
+Install the Network Operator:
+
+.. code-block:: bash
+   :substitutions:
+
+   helm install network-operator nvidia/network-operator \
+     -n nvidia-network-operator \
+     --create-namespace \
+     --version |network-operator-version| \
+     --set sriovNetworkOperator.enabled=true \
+     --wait
+
+Verify the installation:
+
+.. code-block:: bash
+
+   kubectl -n nvidia-network-operator get pods
+
+================================
+Overview of Quickstart Use Cases
+================================
+
+This quick start guide covers five essential networking configurations for different computational requirements:
+
+.. list-table::
+   :widths: 20 25 20 30
+   :header-rows: 1
+
+   * - **Use Case**
+     - **Purpose**
+     - **Performance Requirements**
+     - **Applications**
+   * - `SR-IOV Network with RDMA`_
+     - High-performance networking with hardware acceleration
+     - • >10 Gbps throughput
+       • <1μs latency
+       • Dedicated VF resources
+     - HPC simulations, distributed ML training, financial trading
+       
+       *Keywords: SR-IOV, RDMA, HPC, low-latency, VF isolation*
+   * - `Host Device Network with RDMA`_
+     - Direct hardware access for legacy applications
+     - • Raw device control
+       • Exclusive hardware access
+       • Minimal CPU overhead
+     - Legacy HPC codes, specialized protocols, DPDK applications
+       
+       *Keywords: host-device, PCI-passthrough, direct-access, exclusive-access*
+   * - `IP over InfiniBand with RDMA Shared Device`_
+     - InfiniBand networking with shared RDMA resources
+     - • >50 Gbps bandwidth
+       • Parallel I/O workloads
+       • Shared device efficiency
+     - Distributed storage, data analytics, scientific computing
+       
+       *Keywords: InfiniBand, IPoIB, shared-device, high-bandwidth*
+   * - `MacVLAN Network with RDMA Shared Device`_
+     - Network isolation with shared RDMA capabilities
+     - • Multi-tenant segmentation
+       • 10+ pods per node
+       • Moderate throughput
+     - Cloud-native HPC, microservices, multi-tenant ML
+       
+       *Keywords: MacVLAN, multi-tenant, network-segmentation, resource-sharing*
+   * - `SR-IOV InfiniBand Network with RDMA`_
+     - Virtualized InfiniBand with hardware acceleration
+     - • >100 Gbps bandwidth
+       • Hardware acceleration
+       • Isolated IB partitions
+     - Large-scale HPC clusters, AI/ML training, research computing
+       
+       *Keywords: SR-IOV, InfiniBand, hardware-acceleration, ultra-high-bandwidth*
+
+===================
+Deployment Examples
+===================
+
+.. _sriov-network-with-rdma-deployment:
+
+SR-IOV Network with RDMA
+=========================
+
+**Step 1**: Create NicClusterPolicy
+
+.. literalinclude:: ../_build/examples/sriov-network-rdma/10-nicclusterpolicy.yaml
```

Comment:
First time we use `literalinclude`.
Let's ask Rula for a dry run of this in brighspot.

Reply:
Works fine!